### PR TITLE
New Brig for Security

### PR DESCRIPTION
**Pull request by Ranzarth**
# New Brig for Security

### Purpose
Redesigned the entire brig for added functionality for sec and the rest of the station.

### Notes:
HoS office needs an APC, and lights in cells do not work currently, will fix these in a later pr.

### CODE REVIEW

**Comment by ghost** [comment]
How about some comfy pictures?

EDIT: Oh yes that's very sexy, and it will fix all 5 officers rushing for the L6 instantly. Looks very brig-like... That chair in the interrogation room, it's between tiles? How are you going to sit on it?

File: .gitignore [hunk 1]
```diff
@@ -101,3 +101,4 @@ UnityProject/Assets/StreamingAssets/admin/admins.txt
 UnityProject/Assets/StreamingAssets/admin/admins.txt.meta
 UnityProject/Assets/StreamingAssets/admin/banlist.json
 UnityProject/Assets/StreamingAssets/admin/banlist.json.meta
+*.meta
```
Comment (DooblyNoobly) [inline .gitignore]: ah no. Remove this gitignore. metas are needed to make unity work

**Review by DooblyNoobly — CHANGES_REQUESTED** [review]

**Comment by Ranzarth** [comment]
The chair will just auto assign itself to the nearest tile.

**Comment by fomalsd** [comment]
oh.. could you please exclude all the metafiles that aren't directly relevant to this PR?

**Comment by fomalsd** [comment]
that brig looks cool AF though, looks like it's the largest department on the station now haha

**Comment by Ranzarth** [comment]
Indeed, and I was not able to fit every room required, it still needs an execution room and a permanent prison area, and I will have to add a detective's office nearby, which I already found a spot for.

**Comment by DooblyNoobly** [comment]
I'll merge this and fix remaining issues. Thank you!

**Pull request merged**